docs/web/sdk-components-commands-operations.mdx

```diff
@@ -96,6 +104,20 @@ export default class CustomModel extends ComponentModelBase {
 }
 ```
 
+</TabItem>
+<TabItem value="registration">
+
+```ts
+export default function (registry: LibraryRegistry) {
+    ...
```

Comment:
Let's avoid using the `...` if possible. `// ...` is preferred if needed as it is syntactically valid.

Reply:
Commented out!